README.md

```diff
@@ -1,12 +1,19 @@
-![Geode Logo](title.png)
+<p align="center">
+	<img src="/title.png" />
+	<h3 align="center">
+		<a href="https://geode-sdk.org">Home page</a>
+	</h3>
+</p>
 
-### [Home page](https://geode-sdk.org)
+<p align="center"><b>Geode</b> is a <a href="https://store.steampowered.com/app/322170/Geometry_Dash/">Geometry Dash</a> <b>mod loader</b> and <b>modding SDK</b> with a modern approach towards mod development.</p>
```

Comment:
why replace the markdown link with an <a href>

Reply:
Make it centered. That isn't possible with Markdown. Plus it would look a bit jarring if not centered.